Source/CardStackController.m

```diff
@@ -325,6 +325,31 @@ - (void)cardTitlePanDidFinish:(CardView *)card withVelocity:(CGPoint)velocity {
 
 #pragma mark - Other methods
 
+- (CardView *)cardViewForViewController:(UIViewController *)viewController {
+    CardView *card;
+
+    for (NSUInteger index = 0; index < self.viewControllers.count; index++) {
+        if ([[self.viewControllers objectAtIndex:index] isEqual:viewController]) {
+            card = [self.cards objectAtIndex:index];
+            break;
+        }
+    }
+
+    return card;
+}
+
+- (NSUInteger)indexForViewController:(UIViewController *)viewController {
+    NSUInteger index;
+
+    for (NSUInteger index = 0; index < self.viewControllers.count; index++) {
+        if ([[self.viewControllers objectAtIndex:index] isEqual:viewController]) {
+            break;
```

Comment:
@elvis: if you mean `indexForViewController:', then please have a look at the commit below.